### DISCUSSION TITLE
Hubs Cloud issues with AWS and CDN

### DISCUSSION THREAD

**Discussion by MarcusMorba** [mozilla/hubs#3017] [Hubs Cloud Questions] [answered]
Hi all,
I have some troubles setting up a new Hubs Cloud instance on AWS.
My first setup went well without issues, but after unfortunately "killing" the instance to save costs I now have troubles to get the server running again on AWS.

Trying to run the instance again on the same subdomain (hubs.mucscifi.org) totally failed.
Therefore I tried another subdomain (hubs20.mucscifi.org) which worked, but now I am stuck with enabling CDN which we need to import Sketchfab and Google Poly assets.

**Comment by MarcusMorba** [comment, marked as answer]
Here I am stuck at the moment
![image](https://user-images.githubusercontent.com/22810698/92989241-27ba0280-f4d3-11ea-8118-ebd1cdd8e11d.png)
![image](https://user-images.githubusercontent.com/22810698/92989244-30123d80-f4d3-11ea-8f44-d70405b8cc4f.png)

**Reply by johnshaughnessy** [reply]
I reproduced this scenario on my test instance. 
![image](https://user-images.githubusercontent.com/4072106/93006141-7be1d880-f50d-11ea-85f2-727b02462f69.png)

The issue might be that you are accessing the admin panel, but not actually logged in to an admin account, which means you are not actually able to do anything. We have [an open issue for addressing this confusion](https://github.com/mozilla/hubs/issues/2999).

Until we fix it, one way to check if you really are logged in as admin is to look for an "admin" button on the homepage. If it not there, you are not in an admin account. Alternatively, you can check if you can see any details in the scenes/avatars pages of the admin panel, which will not be available unless you are an admin:
![image](https://user-images.githubusercontent.com/4072106/93006137-6bc9f900-f50d-11ea-9f78-a5038f4d39f4.png)

Hope this helps!

**Reply by MarcusMorba** [reply]
Thanks.

**Reply by MarcusMorba** [reply]
![image](https://user-images.githubusercontent.com/22810698/94282109-bd33a880-ff4f-11ea-9329-196d2a8eeebe.png)
So I think I am Admin

**Comment by MarcusMorba** [comment]
Also when I now try to import a scene into spoke I get the following errors. The proxy-URL displayed was from the previous try to setup the instance and should not exist. 
![image](https://user-images.githubusercontent.com/22810698/92989269-6fd92500-f4d3-11ea-8f4e-283f012594c0.png)

**Reply by johnshaughnessy** [reply]
Let us know if this is still a problem after the admin login / worker setup I mentioned [above](https://github.com/mozilla/hubs/discussions/3017#discussioncomment-77437). I believe we've seen this error message before and I did not know what caused it at the time. Someone else might have the answer.

**Reply by antpb** [reply]
@johnshaughnessy Worth noting, I'm seeing that same behavior on hubs.mozilla.com/spoke

To reproduce, create a new scene and add an image element, you should immediately see the default error thumbnail and a nasty error in your console.

**Reply by johnshaughnessy** [reply]
Thanks for the report. I have reproduced the issue but haven't been able to diagnose it yet

**Reply by robinkwilson** [reply]
@antpb Now that it's a different issue you're seeing other than the cloudflare workers -- I'm going to mark @johnshaughnessy 's as the answer. 

Did you add a **Sketchfab API** key to your admin panel? You'll need to be logged in as an admin for those API keys to save. Admin Panel > Server Settings > API Keys. I believe there was another discussion thread specifically about this error. 

If you have more problems, please create another discussion thread specific to the new issue! Thank you!!

**Reply by MarcusMorba** [reply]
Yes - Sketchfab API key is added. Meanwhile also Sketchfab and Google Poly are working like expected.

**Comment by MarcusMorba** [comment]
thanks for all the help - will try it now. Sorry I was not able to check your feedback earlier.
